ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -252,3 +429,38 @@ extension CommandBar: UIScrollViewDelegate {
         updateShadow()
     }
 }
+
+// MARK: - CommandBarItemDelegate
+
+extension CommandBar: CommandBarItemDelegate {
```

Comment:
Any state change will result in the button to update the state so will it be easier to have one function that handles all stage changes? Or could we have separate functions to update the button's enabled, selected, title, image, etc. properties?

Reply:
Hmm, I'm going to give this a little more thought and then update this PR accordingly.

Reply:
Probably just having one delegate method that all `CommandBarItem` properties call would be the best and easiest right now.